shell/cc-window.c

```diff
@@ -117,6 +117,8 @@ static gboolean cc_window_set_active_panel_from_id (CcShell      *shell,
 
 static gint get_monitor_height (CcWindow *self);
 
+static void update_small_screen_settings (CcWindow *self);
```

Comment:
Not sure you still need this - no need for another round of review though for it.